cmd/flowrunner/main.go

```diff
@@ -21,6 +21,10 @@ import (
 	uuid "github.com/satori/go.uuid"
 )
 
+const (
+	outputIndent string = "    "
```

Comment:
Unrelated but I formatted all the JSON files in a prev PR to use 4 space indentation, so need to update this to not rewrite them with 2 spaces